### PR TITLE
Fix widget position problem

### PR DESCRIPTION
**Pull request by jordanSu**
The layout problem in **encode** and **send packets** page is fixed.
The functionality should also work now.

### CODE REVIEW

**Comment by Still34** [comment]
Looks good to me, though you might want to open branches according to standard conventions like `patches/fix-widget`.

**Comment by Still34** [comment]
Scratch that, the window gets screwed up if you scale it below its intended resolution/size. Perhaps have a predefined minimum size?
image

File: data/screens/encode.kv [hunk 1]
```diff
@@ -1,9 +1,9 @@
-#:import random random.random
+#:kivy 1.8.0
```
Comment (Still34) [inline data/screens/encode.kv:1]: Is it possible to define a minimum resolution?

**Review by Still34 — CHANGES_REQUESTED** [review]

File: data/screens/encode.kv (hunk 1 above)
Comment (l3ouu4n9) [reply data/screens/encode.kv:1]: > Is it possible to define a minimum resolution?

I can handle that.

**Pull request merged**